fbw-a32nx/src/wasm/systems/a320_systems/src/pneumatic.rs

```diff
@@ -1247,21 +1605,25 @@ pub struct PackComplex {
     pack_container: PneumaticPipe,
     exhaust: PneumaticExhaust,
     pack_flow_valve: DefaultValve,
+    pack_inlet_pressure_sensor: PressureTransducer,
 }
 impl PackComplex {
-    fn new(context: &mut InitContext, engine_number: usize) -> Self {
+    fn new(context: &mut InitContext, engine_number: usize, powered_by: ElectricalBusType) -> Self {
         Self {
             engine_number,
             pack_flow_valve_id: context.get_identifier(Self::pack_flow_valve_id(engine_number)),
             pack_flow_valve_flow_rate_id: context
                 .get_identifier(format!("PNEU_PACK_{}_FLOW_VALVE_FLOW_RATE", engine_number)),
             pack_container: PneumaticPipe::new(
-                Volume::new::<cubic_meter>(2.),
+                Volume::new::<cubic_meter>(1.),
                 Pressure::new::<psi>(14.7),
                 ThermodynamicTemperature::new::<degree_celsius>(15.),
             ),
             exhaust: PneumaticExhaust::new(0.3, 0.3, Pressure::new::<psi>(0.)),
             pack_flow_valve: DefaultValve::new_closed(),
+            // TODO: Check electrical supply of this
+            // I'm pretty sure PACK 1 is DC 1 and PACK 2 is DC 2, but happy to be corrected
```

Comment:
I think this comment is better placed where the `PackComplex` is instantiated (i.e. in `A320Pneumatic`).